extensions/git/src/model.ts

```diff
@@ -241,9 +259,10 @@ export class Model {
 			}
 
 			const config = workspace.getConfiguration('git');
-			const ignoredRepos = new Set(config.get<Array<string>>('ignoredRepositories'));
 
-			if (ignoredRepos.has(rawRoot)) {
+			const ignoredRepos = (config.get<Array<string>>('ignoredRepositories') || []).map(x => Uri.file(x).fsPath);
```

Comment:
That change is related to #33498
Without that change, if user type 'C:/asd', it won't ignore repo c:/asd in case insensitive file systems.